tests/BallotParserTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace PivotLibre\Tideman;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class BallotParserTest
+ * Presently, the BallotParser is just a thin wrapper around the CandidateRankingParser. We can safely perform minimal
+ * testing here because most functionality is extensively tested in CandidateRankingParserTest.
+ * @package PivotLibre\Tideman
+ */
+class BallotParserTest extends TestCase
+{
+    private $instance;
+    private $alice;
+    private $bob;
+    private $claire;
+
+    public function setUp()
+    {
+        $this->instance = new BallotParser();
+        $this->alice = new Candidate("A");
+        $this->bob = new Candidate("B");
+        $this->claire = new Candidate("C");
+    }
+
+    public function testGoodBallot()
+    {
+        $expected = new Ballot(
+            new CandidateList($this->alice),
+            new CandidateList($this->bob),
+            new CandidateList($this->claire)
+        );
+
+        $actual = $this->instance->parse("A>B>C");
+        $this->assertEquals($expected, $actual);
+    }
+
+    public function testThatAsteriskBreaks()
+    {
+        //BallotParser should not parse NBallot text
+        $this->expectException(\InvalidArgumentException::class);
+        $this->instance->parse("1*A>B>C");
```

Comment:
I'm not convinced of the value of the multiplication syntax, since I imagine most inputs will be generated automatically (by pivot or by testing tools), but if you feel it's useful, I'm not opposed.  Is this something that you want implemented in the verification tools (in both Python and JavaScript) that I've implemented?  For example, this: https://github.com/pivot-libre/ballot-gen/blob/master/graph_parse.py.  Or this: https://github.com/pivot-libre/pivot/blob/master/public/js/results.js#L266.

Reply:
Since I believe that both machine-generated and manually-constructed tests are valuable, I believe it is useful to support this syntax for manually-constructed sets of test ballots.

I do not anticipate that either of the verification tools will need to support this syntax.

Reply:
Sounds good, just wanted to make sure we weren't creating unanticipated work down the road.  I would check this in, then.